sites/public/src/pages/search-demo.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react"
+import Head from "next/head"
+import Layout from "../layouts/application"
+import { MetaTags } from "../components/shared/MetaTags"
+import { PageHeader, t } from "@bloom-housing/doorway-ui-components"
+import { FormOption } from "../components/listings/search/ListingsSearchForm"
+import { ListingsSearchCombined } from "../components/listings/search/ListingsSearchCombined"
+import { getListingServiceUrl } from "../lib/helpers"
+import { runtimeConfig } from "../lib/runtime-config"
+
+type SearchTestProps = {
+  listingsEndpoint: string
+  googleMapsApiKey: string
+  initialSearch?: string
+  bedrooms: FormOption[]
+  bathrooms: FormOption[]
+  locations: FormOption[]
+}
+
+// Input values/options below are passed into the form to make it easier to
+// reuse it in multiple places.  This may not ultimately be necessary, but it's
+// easier to add it in at the beginning than it is to try to add make the change
+// later.
+
```

Comment:
See comment above.  I will add, however, that based on the landing page mocks seen today, it's probably really important that we keep these outside and inject them in.  It would be way easier to add a condition that says "if bathrooms aren't present then don't show the options" than it would be to create a completely different search box, as long as they aren't so radically different in appearance that you would need to make a completely different one anyway.